lib/import/colorectal/providers/leeds/leeds_handler_new_colorectal.rb

```diff
@@ -0,0 +1,503 @@
+        # Leeds importer for colorectal for post 2025 files
+        class LeedsHandlerNewColorectal < Import::Germline::ProviderHandler
+          include Import::Helpers::Colorectal::Providers::Rr8::Constants
+
+          def process_fields(record)
+            # check if should process record from Other Cancer file
+            return unless should_process?(record)
+
+            genocolorectal = Import::Colorectal::Core::Genocolorectal.new(record)
+            genocolorectal.add_passthrough_fields(record.mapped_fields,
+                                                  record.raw_fields,
+                                                  PASS_THROUGH_FIELDS,
+                                                  FIELD_NAME_MAPPINGS)
+
+            populate_variables(record)
+            process_test_scope(genocolorectal)
+            setup_derived_values(record)
+            genotypes = []
+
+            if genocolorectal.full_screen?
+              add_fs_moleculartestingtype(genocolorectal, record)
+              @genes_panel = genes_panel
+              res = process_fs_rec(genocolorectal, record, genotypes)
+            elsif genocolorectal.targeted?
+              add_targ_moleculartestingtype(genocolorectal)
+              res = process_targ_rec(genocolorectal, record, genotypes)
+            end
+
+            res.each { |cur_genotype| @persister.integrate_and_store(cur_genotype) }
+          end
+
+          def should_process?(record)
+            file_name = @batch.original_filename
+            return true unless file_name =~ /Other/ix
+
+            fields = record.raw_fields
+            return false unless fields['moleculartestingtype'] == 'Familial'
+
+            rep_scan  = fields['report']&.scan(MMR_GENE_REGEX)
+            diag_scan = fields['diagnosis_report']&.scan(MMR_GENE_REGEX)
+            has_genes = rep_scan&.any? || diag_scan&.any?
+
+            return false unless has_genes
+            return false if fields['diagnosis_report'] =~ /ataxia/i
+            return false if fields['codingdnasequencechange'] =~ /BRCA/i
+
+            true
+          end
+
+          def populate_variables(record)
+            populate_raw_field_variables(record)
+            initialize_processing_variables
+          end
+
+          def populate_raw_field_variables(record)
+            @report = record.raw_fields['report']
+            @moltestingtype = record.raw_fields['moleculartestingtype']
+            @value1 = record.raw_fields['proteinimpact']
+            @value12 = record.raw_fields['zygosity']
+            @report_result = record.raw_fields['genotype']
+            @value2 = record.raw_fields['gene']
+            @result = record.raw_fields['codingdnasequencechange']
+            @diag_report = record.raw_fields['diagnosis_report']
+            @comment = record.raw_fields['variantpathclass']
+            @test = record.raw_fields['karyotypingmethod']
+          end
+
+          def initialize_processing_variables
```

Comment:
Unusual whitespace here. I like the way the `= nil` are aligned, could the `@zygosity` be aligned?